tests/runner.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+if __name__ == '__main__':
+
+    import argparse
+    import sys
+
+    from suite import TestSuite
+
+    parser = argparse.ArgumentParser("The simple regression test suite.")
+
+    parser.add_argument(
+        "config_file",
+        nargs=1,
+        type=str,
+        help="The main configuration file"
```

Comment:
We could add here a default value to the main configuration file (that should be the same in most of the cases). What do you think ?

Reply:
That won't hurt imho.